custom_components/homewhiz/appliance_config.py

```diff
@@ -0,0 +1,124 @@
+from dataclasses import dataclass
+from typing import List, Optional
+
+
+@dataclass
+class ApplianceSubprogramOverride:
+    allowedValueIndices: Optional[List[int]]
```

Comment:
Other code in this project use optional annotation without import from typing—e.g. `int | None`. Would be nice to have this consistent. Since we basically use the same Python version as Homeassistant, we could ditch all typing imports I think.

Reply:
Optional notation instead of `| None` is required by https://github.com/konradhalas/dacite#optional-fields unfortunately